README.md

```diff
@@ -96,34 +110,38 @@ cd SmartSim-Scaling
 pip install -r requirements.txt
 ```
 
+> Note that if you are using a Cray machine, you will need to run `CC=cc CXX=CC pip install -r requirements.txt`.
+
 Lastly, the C++ applications themselves need to be built. To complete this, 
-one CMake edit is required. Navigate to your respective tests `CMakeLists.txt` file.
-If you need assistance finding the file path, please reference the chart below.
-Near the top of `CMakeLists.txt`, on line
-`set(SMARTREDIS "../../SmartRedis" CACHE PATH "Path to SmartRedis root directory")`, 
-change the path to the ``SMARTREDIS`` variable to
-the top level of the directory where you built or installed the SmartRedis library. 
+one CMake edit is required. When running `cmake ..`, 
+change the path to the ``SMARTREDIS`` variable to the top level of the directory 
+where you built or installed the SmartRedis library using the ``-DSMARTREDIS`` flag.
+An example of this is shown below.
+
+All tests can be built by running
+
+```bash
+  cd <language name>-<test name> # ex. cpp-inference for the cpp inference tests
+  mkdir build && cd build
+  cmake .. -DSMARTREDIS=/path/to/SmartRedis
+  make
+```
+
+The app locations are shown below: 
 
 1. Inference
    - `cpp-inference/CMakeLists.txt`
+   - `fortran-inference/CMakeLists.txt`
 2. Throughput
    - `cpp-throughput/CMakeLists.txt`
 3. Data Aggregation
    - `cpp-data-aggregation/CMakeLists.txt`
    - `cpp-py-data-aggregation/db/CMakeLists.txt`
    - `cpp-py-data-aggregation/fs/CMakeLists.txt`
 
-After the cmake edit, all tests can be built by running
-
-```bash
-  cd <language name>-<test name> # ex. cpp-inference for the cpp inference tests
-  mkdir build && cd build
-  cmake ..
-  make
-```
-
 > Note that there are three different `CMakeLists.txt` files for the Data Aggregation tests.
-You will need to build per Data Aggregation scaling test.
+You will need to build per Data Aggregation scaling test. This is the same for the 
```

Comment:
Note quite sure what 'This' is referring to. The installation instrucitons? If so, clarify